docs/howto/determining-packages.md

```diff
@@ -0,0 +1,172 @@
+# Determining Packages in Pangeo Docker Images
+
+For deployments of Pangeo Docker Images using JupyterHub or Binder it can be sometimes unclear to the user exactly which image and tag is being used. Here are the images available as Pangeo Docker Images:
+
+| Image           | Description                                   | 
+|-----------------|-----------------------------------------------|
+| [base-notebook](base-notebook/packages.txt) | minimally functional image for pangeo hubs | 
+| [pangeo-notebook](pangeo-notebook/packages.txt) | base-notebook + core earth science analysis packages |
+| [pytorch-notebook](pytorch-notebook/packages.txt) | pangeo-notebook + GPU-enabled pytorch |
+| [ml-notebook](ml-notebook/packages.txt) | pangeo-notebook + GPU-enabled tensorflow2 | 
+| [forge](ml-notebook/packages.txt) | pangeo-notebook + [Apache Beam](https://beam.apache.org/) support| 
+
+Each image is also assigned a tag of the form `YYYY.MM.DD` indicating when it was last updated.
+
+*Click on the image name in the table above for a current list of installed packages and versions for the most recent tag of each image*
+
+## How to find my current image tag
+
+The environment variable `JUPYTER_IMAGE_SPEC` will tell you what Pangeo Docker Image is currently in use.
+
+### Using the command line with Bash
+
+```bash
+echo JUPYTER_IMAGE_SPEC: $JUPYTER_IMAGE_SPEC
+
+tmp_array=($(echo $JUPYTER_IMAGE_SPEC | tr "/:" "\n"))
+image_name=${tmp_array[1]}
+image_tag=${tmp_array[2]}
+
+echo image_name: $image_name
+echo image_tag: $image_tag
+```
+
+### Using a Jupyter notebook with Python
+
+```python
+import os
+image_spec = os.environ['JUPYTER_IMAGE_SPEC']
+print('JUPYTER_IMAGE_SPEC:', image_spec)
+
+_, image_name, image_tag = image_spec.replace('/',':').split(':')
+
+print('image_name:', image_name)
+print('image_tag:', image_tag)
+```
+
+## How to find the list of available image tags:
+
+Using GitHub the web site, the list of select a particular tag from the branches/tag drop down menu.
+
+## How to finding list of packages in any given tag
+
+Each image has an `packages.txt` file that lists exactly which packages and versions installed in the image.
+
+### Using GitHub website
+
+Using GitHub the web [site](https://github.com/pangeo-data/pangeo-docker-images), select a particular tag from the branches/tag drop down menu.  Navigate to the image and select the `packages.txt` file.
+
+### Using the command line with Bash
```

Comment:
IMO, just focusing on using the GitHub UI is good enough.

Reply:
I think it depends on the user and what other tools they are used to using.